pnparser/helpers.py

```diff
@@ -0,0 +1,42 @@
+import re as Regex
+from bs4 import Tag
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Iterator
+    from re import Match
+
+
+EXCEPTIONS: 'list[str]' = ["and", "or", "the", "a", "of", "in", "ARAM", "QoL"]
+
+
+class Helper:
+    @staticmethod
```

Comment:
make this a classmethod and then EXCEPTIONS can be an attribute of Helper class

Reply:
I don't know if I understood this properly, I changed it to be non-static and moved `EXCEPTIONS` to an constant inside the class, is that it?